app/views/users/show.html.erb

```diff
@@ -80,24 +80,11 @@ See COPYRIGHT and LICENSE files for more details.
       <p>
         <%=t(:label_reported_work_packages)%>: <%= @user.reported_work_package_count %>
       </p>
-      <div id="activity">
-        <% @events_by_day.keys.sort.reverse.each do |day| %>
-          <h4><%= format_activity_day(day) %></h4>
-          <ul class="generic-list">
-            <% @events_by_day[day].sort {|x,y| y.event_datetime <=> x.event_datetime }.each do |e| -%>
-              <li class="<%= e.event_type %>">
-                <div class="title">
-                  <%= icon_wrapper("icon-context icon-#{event_icon(e)}", e.event_name) %>
-                  <span class="time"><%= format_time(e.event_datetime, false) %></span>
-                  <%= content_tag('span', h(e.project), class: 'project') %>
-                  <%= link_to format_activity_title(e.event_title), e.event_url %>
-                </div>
-                <div class="description">
-                  <%= truncate_formatted_text(e.event_description) %>
-                </div>
-              </li>
-            <% end -%>
-          </ul>
+      <div id="activity" class="op-activity-days">
+        <% @events_by_day.sort_by { |day, _events| day }.reverse_each do |day, events| -%>
+          <h4 class="op-activity-days--header"><%= format_activity_day(day) %></h3>
+
+          <%= render(ActivityListComponent.new(events:, journals_by_id: @journals_by_id, display_user: false)) %>
```

Comment:
I am probably getting greedy but we could move the 'activity by day' functionality into its own view component. The `id="activity"` can probably be remove alltogether so the only difference between this code here and the one in the `activities/index.html` seems to be the `display_user: false` flag.
Isn't a must though, just an opportunity.

Reply:
Should be ok now with latest commit.